s3.py

```diff
@@ -0,0 +1,230 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# based on https://github.com/mozilla/autophone/blob/master/s3.py
```

Comment:
Yeah, I think that's the eventual conclusion. Regarding Treeherder submission overall, 
there is a lot of overlap between this and firefox-media-tests, Henrik's firefox-ui-tests and Autophone. There are plans to discuss how to avoid repetition and make Treeherder submission easier.